app/interactors/headlines/parse_security_headers.rb

```diff
@@ -49,12 +49,19 @@ def header_class(header)
     end
 
     def connection
-      Faraday.new(url: "http://#{context.url}", headers: { accept_encoding: "none" }) do |builder|
+      Faraday.new(url: "http://#{context.url}", headers: request_headers, ssl: { verify: false }) do |builder|
         builder.request :url_encoded
```

Comment:
Why do we need to disable SSL verification?
I think we need to rethink how we are doing the scan for http v/s https sites a bit. I do not think disabling the SSL verification is a good idea.

There was also a good suggestion on the open site about it - https://open.srcclr.com/t/additional-headers-additional-features/58